docs/reporting.md

```diff
@@ -57,42 +57,49 @@ The general information can be useful for MFTF contributors, but can be ignored
 
 #### Passed tests
 
 The next chunk of the log reports about test execution of the first test:
 
 ```terminal
-AdminLoginTestCest: Admin login test
-Signature: Magento\AcceptanceTest\_default\Backend\AdminLoginTestCest:AdminLoginTest
-Test: tests/functional/Magento/FunctionalTest/_generated/default/AdminLoginTestCest.php:AdminLoginTest
+AdminLoginSuccessfulTestCest: Admin login successful test
+Signature: Magento\AcceptanceTest\_default\Backend\AdminLoginSuccessfulTestCest:AdminLoginSuccessfulTest
+Test: tests/functional/tests/MFTF/_generated/default/AdminLoginSuccessfulTestCest.php:AdminLoginSuccessfulTest
 Scenario --
-I am on page "/admin/admin"
-I fill field "#username","admin"
-I fill field "#login","123123q"
-I click ".actions .action-primary"
-I wait for page load 30
-I close admin notification
-I see in current url "/admin/admin"
-PASSED
+[loginAsAdmin] AdminLoginActionGroup
+  [navigateToAdmin] am on page "/admin/admin"
+  [fillUsername] fill field "#username","admin"
+  [fillPassword] fill field "#login","123123q"
+  [clickLogin] click ".actions .action-primary"
+  [clickLoginWaitForPageLoad] wait for page load 30
+  [clickDontAllowButtonIfVisible] conditional click ".modal-popup .action-secondary",".modal-popup .action-secondary",true
+  [closeAdminNotification] close admin notification 
+[assertLoggedIn] AssertAdminSuccessLoginActionGroup
+  [waitForAdminAccountTextVisible] wait for element visible ".page-header .admin-user-account-text",60
+  [assertAdminAccountTextElement] see element ".page-header .admin-user-account-text"
+[logoutFromAdmin] AdminLogoutActionGroup
+  [amOnLogoutPage] am on page "/admin/admin/auth/logout/"
+ PASSED 
 ```
 
-The running test is `AdminLoginTestCest`, which is `Admin login test` (this text is generated from the test name but with the `Cest` part excluded).
-Its test signature is `Magento\AcceptanceTest\_default\Backend\AdminLoginTestCest:AdminLoginTest` that matches a `className:methodName` format using namespaces.
-A path to the corresponding `Test` is `tests/functional/Magento/FunctionalTest/_generated/default/AdminLoginTestCest.php:AdminLoginTest` (relative to the `acceptance/` directory).
+The running test is `AdminLoginSuccessfulTestCest`, which is `Admin login successful test` (this text is generated from the test name but with the `Cest` part excluded).
+Its test signature is `Magento\AcceptanceTest\_default\Backend\AdminLoginSuccessfulTestCest:AdminLoginSuccessfulTest` that matches a `className:methodName` format using namespaces.
+A path to the corresponding `Test` is `tests/functional/tests/MFTF/_generated/default/AdminLoginSuccessfulTestCest.php:AdminLoginSuccessfulTest` (relative to the `acceptance/` directory).
```

Comment:
I would use the path from the embedded setup

Reply:
fixed